minimesos/src/main/java/com/containersol/minimesos/docker/DockerClientFactory.java

```diff
@@ -3,28 +3,84 @@
 import com.github.dockerjava.api.DockerClient;
 import com.github.dockerjava.core.DockerClientBuilder;
 import com.github.dockerjava.core.DockerClientConfig;
-import org.apache.commons.lang.StringUtils;
+import java.io.File;
+import java.util.Map;
+import java.util.Properties;
+
+import static com.github.dockerjava.core.DockerClientConfig.*;
 
 /**
  * Factory for creating {@link DockerClient}s
  */
 public class DockerClientFactory {
-
     private static DockerClient dockerClient;
+    private static DockerClientConfigBuilder configBuilder = new DockerClientConfigBuilder();
+    private static String dockerHostEnv;
+    private static String dockerCertPathEnv;
+    private static String dockerHostPrp;
+    private static String dockerCertPathPrp;
```

Comment:
These four are using only within a single method and therefore can (should) become local variables:
```
    private static String dockerHostEnv;
    private static String dockerCertPathEnv;
    private static String dockerHostPrp;
    private static String dockerCertPathPrp;
```